provision/ubuntu/kernel-next.sh

```diff
@@ -11,7 +11,7 @@ sudo apt-get install -y --allow-downgrades \
 	cmake libdw-dev git
 
 # Build pahole
-PaholeVer="1.18"
+PaholeVer="1.20"
```

Comment:
Where are we setting `CONFIG_DYNAMIC_FTRACE` as the linked issue suggests?

Reply:
I haven't found this to be necessary contrary to what Lorenz was saying. It could be that it's already enabled in our case. In any case, since this is likely a transient issue in bpf-next, I'd rather just update pahole (since we'll need to do that at some point anyway).

Reply:
But why not do both as your PR description suggests? We are anyway setting a bunch of CONFIGs down below starting @L34.

Reply:
Because we don't need `CONFIG_DYNAMIC_FTRACE` so we might as well not set it.

> But why not do both as your PR description suggests?

Sorry, forgot to update the PR description after I updated the commit description :facepalm: